robmuxinator/robmuxinator.py

```diff
@@ -800,6 +809,11 @@ def main():
             else:
                 port = DEFAULT_PORT
 
+            if "ssh_port" in yaml_hosts[key]:
```

Comment:
please document the new key in https://github.com/4am-robotics/robmuxinator?tab=readme-ov-file#yaml-configuration